### PR TITLE
Add #programming-language

### PR DESCRIPTION
**Pull request by Tyler887**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/master/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/master/docs>

> Popular topic that should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Review by Pakon2543 — COMMENTED** [review]
4AEE18F83AFDEB23

**Comment by Grzegorzkociole** [comment]
V

**Comment by Snowiiii** [comment]
I think that if you use github you already know what a programming language is

File: topics/programming-language/index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+display_name: Programming language
+short_description: A programming language is a language for developing apps and all other sorts of programs.
+
+topic: programming-language
+---
+A programming language is a language for developing apps, games all other sorts of programs. No programming language = plain text.
```
Comment (kenyonj) [inline topics/programming-language/index.md:7]: This bit is a little confusing to me. Can it be reworded? I am not sure what "No programming language = plain text" means either.

File: topics/programming-language/index.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+display_name: Programming language
+short_description: A programming language is a language for developing apps and all other sorts of programs.
```
Comment (kenyonj) [inline topics/programming-language/index.md:3]: I think we could be a bit more descriptive about how a programming language is something that can be used to write instructions for a computer to interpret, or something along those lines.

**Review by kenyonj — CHANGES_REQUESTED** [review]
As it sits right now, I think the additions aren't very helpful to the reader. Perhaps you can take another crack at adding more information/context?

**Review by KenyanFace — COMMENTED** [review]
Good work

**Review by ss-pixelporksword — COMMENTED** [review]
Nice!

**Comment by Tyler887** [comment]
> Nice!

Stop spamming around here. I think the @github org would try to delete spam from this repo. Remember to follow the [Code of Conduct for this GitHub project](https://github.com/github/explore/blob/6605bee189af15c75ab0809b29658179d939bc7a/CODE_OF_CONDUCT.md).

**Pull request closed without merging**